CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# XXXXXXX
+
+## ⭐ Features
+
+- The function `fun toString(): String` was added to all path types. It returns the textual representation of the type.
+
```

Comment:
This file is updated based on the GitHub release notes, which are generated automatically by a bot from the pull requests, so this isn't needed.

```suggestion
```